src/lib/parts.ts

```diff
@@ -549,6 +539,21 @@ export class BooleanAttributePart implements Part {
   }
 }
 
+const namespaces: {[name: string]: string | undefined} = {
+  'xlink': 'http://www.w3.org/1999/xlink',
+  'xml': 'http://www.w3.org/XML/1998/namespace',
+} as const;
+
+const getAttrName = (name: string): [string|null, string] => {
+    const colonIndex = name.indexOf(':');
+    if (colonIndex === -1) {
+      return [null, name];
+    }
+    const shorthand = name.slice(0, colonIndex);
+    const namespace = namespaces[shorthand] || null;
```

Comment:
So if the namespace is unknown we just silently ignore it?

I'm not sure of a better option, but that's a bit unfortunate

Reply:
Not sure what else we can do here. I tried a few different ways to get the actual namespace URI from the DOM, but they all fail. Hardcoding the common prefixes or punting completely are the only options left. Since we've had multiple reports on this, I guess we should do it.